templates/partials/answers/textfield.html

```diff
@@ -32,6 +32,7 @@
     "ariaLimitedResults": "Results have been limited to 10 suggestions. Type more characters to improve your search.",
     "typeaheadData": answer.suggestions_url
   }) %}
+  {% set config = config | setAttribute("autocomplete", "off") %}
```

Comment:
Can these no just be added to in the initial config object?

Reply:
No, by default it should be on for a textfield and only switched off when suggestions are used.

Reply:
Ah didn't see the `if`